bundler/dust.js

```diff
@@ -21,7 +21,7 @@ var spud = require('spud');
 var freshy = require('freshy');
 var loopalo = require('../lib/loopalo');
 //import dust here to get around issue in freshy where freshy.freshy doesn't properly check for an existing copy
-var doost = require('dustjs-linkedin');
+require('dustjs-linkedin');
```

Comment:
I think that first require just isn't needed at all, now. `freshy` should work fine even when the library hasn't been loaded yet.